server/graphql/mutations/orders.js

```diff
@@ -238,6 +239,10 @@ export function createOrder(order, loaders, remoteUser) {
     })
     .catch(e => {
       debugOrder("createOrder mutation error: ", e);
+      if (!orderCreated.processedAt) {
+        // TODO: Order should be updated with data JSON field to store the error to review later
+        orderCreated.update({ status: status.ERROR });
+      }
       throw e;
```

Comment:
Do you need to throw the error as you're already inside the catch?

Reply:
I believe it was thrown again to bubble it back up to the graphql response, otherwise the frontend never sees the error.

Reply:
got it! LGTM then..